explorer/models/commonRegistryUtils.ts

```diff
@@ -93,6 +92,12 @@ export async function getTags(registryUrl: string, repositoryName: string, crede
     return tagInfos;
 }
 
+export async function getAssociatedTags(registryUrl: string, repositoryName: string, digest: string, credentials?: RegistryCredentials): Promise<string[]> {
+    let result = await registryRequest<{ manifest: Manifest }>(registryUrl, `/acr/v1/${repositoryName}/_manifests/${digest}`, credentials);
```

Comment:
ACR-specific

Reply:
Same question. Do I need to make a new file for this ones?

Reply:
Same answer.